CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+## Coding style
+
+* HOL source code
+  - Spaces, no tabs
+  - No unicode
+  - `snake_case` (e.g. `bir_number_of_mem_splits_def`)
+
+
+
+## Branch policy
+
+### tags
+
+tags should have as many **completed features** as possible:
+ - no cheat
+ - must correctly compile
+ - self tests must succeed
+ - code should be tested
+
+Follow these instructions whenever you merge to master:
+  - `grep` for "cheat"
```

Comment:
Is that still true, now that master is our dev branch, and that we should merge often?

Reply:
let's fix a test that checks exactly this, and i would say yes

Reply:
How do you want to check that? Is there a wait to forbid cheating with Holmake?

Reply:
no, but it could grep cheat 😄

Reply:
Yeah but that doesn't work, just try it on your local repository. However, you can grep the build logs, Holmake writes "CHEATED" when that's the case.

Reply:
then this means grepping sml files only